tools/config/seed.config.ts

```diff
@@ -21,7 +21,8 @@ export class SeedConfig {
   ENABLE_HOT_LOADING   = argv['hot-loader'];
   HOT_LOADER_PORT      = 5578;
 
-  BOOTSTRAP_MODULE     = this.ENABLE_HOT_LOADING ? 'hot_loader_main' : 'main';
+  BOOTSTRAP_DIR        = 'app';
```

Comment:
Here is another "issue" or "proposal" of sorts. The styleguide states
> Do put bootstrapping and platform logic for the app in a file named main.ts.

In the folder structure examples, this file resides in `src/app` (`src/app/main.ts`).
In the seed the `main.ts` this file currently resides in `src/client` (`src/client/main.ts`).
To conform to the styleguide would it be better to move this to `src/client/app`? (`src/client/app/main.ts`). This is what i went for in this first commit here.

I'm looking for input on that too :)